### ISSUE TITLE
Buggy redirects on doc.rust-lang.org

### ISSUE THREAD

**Issue by kpcyrd** [steveklabnik/rustdoc#198]
I'm not sure if this is the correct repo to report this bug, a hint to the correct repo would be appreciated.

I've noticed a buggy redirect on doc.rust-lang.org:

`https://doc.rust-lang.org/doc.example.com` redirects to `https://doc.rust-lang.org.example.com`.

**Comment by Mark-Simulacrum** [comment]
cc @alexcrichton -- this seems like it's unrelated to rustdoc

**Comment by steveklabnik** [comment]
Yes, this would be an infra issue, which normally is filed against the rust repo. Re-filing probably isn't worth it though, since infra people have already seen it :)
